index.html

```diff
@@ -111,9 +111,8 @@ <h3>Concept and Initial Report</h3>
     <select multiple class="filter--scope">
       <option value="Regional" selected>Regional</option>
       <option value="Citywide" selected>Citywide</option>
-      <option value="City Center/Business District" selected>
-        City Center/Business District
-      </option>
+      <!-- prettier-ignore -->
+      <option value="City Center/Business District" selected>City Center/Business District</option>
```

Comment:
Quick fix to keep the text aligned. Otherwise, Prettier makes this multiple lines, which adds a space in front of "City Center/Business District" in the filter. Is there a better fix for this?

Reply:
Hm, no I don't know of a better approach.